app/controllers/miq_policy_rsop_controller.rb

```diff
@@ -93,7 +93,7 @@ def rsop_option_changed
     if params[:filter_value]
       @sb[:rsop][:filter_value] = params[:filter_value] == "<Choose>" ? nil : params[:filter_value]
     end
-    @rsop_events = MiqEventDefinitionSet.all.collect { |e| [e.description, e.id.to_s] }.sort
+    @rsop_events = MiqEventDefinitionSet.all.collect { |e| [N_(e.description), e.id.to_s] }.sort
```

Comment:
Think this should be
```suggestion
    @rsop_events = MiqEventDefinitionSet.all.collect { |e| [_(e.description), e.id.to_s] }.sort
```

Reply:
You're gonna love how i did do that originally and it wouldn't translate.

Reply:
That's weird - wonder if it's missing in the catalog?

Reply:
That's the thing, it's _in the catalog_ (or at least all the JSON translation fileS) i looked them up and these strings in the drop down are all already translated.

Reply:
I'm so confused 😖

Reply:
Same, codebase haunted 👻

Reply:
Weird.  Are we ok with this?

Reply:
We found a fix for this, pushing up changes now